### DISCUSSION TITLE
AttributeError 'Command' object has no attribute 'run'

### DISCUSSION THREAD

**Discussion by BenCheung0422** [Rapptz/discord.py#6025] [Q&A]
used the following code but it show AttributeError 'Command' object has no attribute 'run'

`bot.run(jdata['Token'])`

**Comment by Harmon758** [comment]
You have `bot` defined as an instance of [`Command`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Command), rather than [`Bot`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Bot).
Without seeing the rest of your code, there's nothing else that can be determined to help you further.
